ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -73,6 +92,24 @@ class CellEditorOverlay extends Component {
     }
   }
 
+  handleSetRange(e) {
+    const {min, max} = e.target.form
+    const {axes} = this.state
+
+    this.setState({
+      axes: {...axes, y: {...axes.y, bounds: [min.value, max.value]}},
+    })
+    e.preventDefault()
+  }
+
+  handleSetLabel(e) {
+    const {label} = e.target.form
+    const {axes} = this.state
+
+    this.setState({axes: {...axes, y: {...axes.y, label: label.value}}})
+    e.preventDefault()
```

Comment:
I can easily deconstruct each of the nodes based on their `names`.  Also, in the future if we want to have a separate submission / validation for them a form will make that easy.